x-pack/plugins/cloud_integrations/cloud_chat/public/plugin.tsx

```diff
@@ -92,12 +92,11 @@ export class CloudChatPlugin implements Plugin<void, void, CloudChatSetupDeps, C
 
   public stop() {}
 
-  private async setupChat({ cloud, http, security }: SetupChatDeps) {
+  private async setupChat({ cloud, http }: SetupChatDeps) {
     const { isCloudEnabled, trialEndDate } = cloud;
     const { chatURL, trialBuffer } = this.config;
 
     if (
-      !security ||
```

Comment:
I didn't replace it here... I assumed that the server side would already handle the state in the API response. The alternative would be fetching the current user, which might require an extra unnecessary HTTP request.